src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -450,6 +450,18 @@ export default {
 	emits: ['update:checked'],
 
 	computed: {
+		ariaAttrs() {
+			// filter all aria attributes
+			return Object.fromEntries(Object.entries(this.$attrs)
+				.filter(([key]) => key.startsWith('aria-')))
+		},
+
+		nonAriaAttrs() {
+			// filter all non-aria attributes
+			return Object.fromEntries(Object.entries(this.$attrs)
+				.filter(([key]) => !key.startsWith('aria-')))
+		},
+
```

Comment:
What I think we could do here, especially because you now can not pass native attributes anymore, is for a "quick fix" do not use `aria-` here but `data-`.

Pass everything to the input except `data-` as data is non functional attributes so we are save not to forward them.

Reply:
That would be the safest solution, I like that idea. But it's still an oversight. 
I'm pretty sure there are tons of other html attributes that would make sense on the outer wrapper :shrug: